daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -38,6 +39,7 @@ object Compiler {
 }
 
 final case class Compiler(packages: PackageId PartialFunction Package) {
+  private val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
```suggestion
  private[this] val logger = LoggerFactory.getLogger(this.getClass)
```
In this case `private[this]` makes a lot of sense. :slightly_smiling_face: